src/main.js

```diff
@@ -122,83 +115,77 @@ var quotes = [
 var savedPosters = [];
 var currentPoster;
 
-// event listeners go here 👇
 window.addEventListener('load', generatePoster);
 randomPosterBtn.addEventListener('click', generatePoster);
 makePosterBtn.addEventListener('click', makePoster);
 savedPosterBtn.addEventListener('click', showSaved);
 showMainBtn.addEventListener('click', showMain);
 backToMainBtn.addEventListener('click', backToMain);
 showMyPosterBtn.addEventListener('click', showMyPoster);
-savePosterBtn.addEventListener('click', savePoster)
-savedPostersGrid.addEventListener('dblclick', deletePoster)
-
-// functions and event handlers go here 👇  
-// (we've provided one for you to get you started)!
-
-
+savePosterBtn.addEventListener('click', savePoster);
+savedPostersGrid.addEventListener('dblclick', deletePoster);
 
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
-}
+};
 
 function generatePoster() {
   currentPoster = new Poster(images[getRandomIndex(images)], titles[getRandomIndex(titles)], quotes[getRandomIndex(quotes)]);
   displayPoster();
-}
+};
 
 function displayPoster() {
   posterSource.src = currentPoster.imageURL;
   posterTitle.innerText = currentPoster.title;
   posterQuote.innerText = currentPoster.quote;
-}
+};
 
 function customPoster() {
```

Comment:
functions should usually have verb naming, something like `buildCustomPoster` or `setCustomPoster`